_posts/2024-11-10-5.10.0-released.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: post
+title: "Luanti 5.10.0 released!"
+author: luatic, grorp, GreenXenith
+description: >-
+  Luanti 5.10.0 has been released with 
+image: /static/blog/5.10.0/cover.webp
+image_credit: Gefüllte Taubenbrust
+forum_topic: https://forum.luanti.org/viewtopic.php?p=440491
+tags:
+  - release
+---
+
+<h2 class="sr-only">Introduction</h2>
+
+Luanti ([formerly Minetest](https://blog.minetest.net/2024/10/13/Introducing-Our-New-Name/)) 5.10.0 has been released
```

Comment:
Links in the excerpt break the homepage card (www.luanti.org) currently. Can get around this by providing an `excerpt` in the front matter without the link and removing `<!-- more -->`

Reply:
The embed is https://blog.minetest.net/embed_latest/ if you want to test locally

Reply:
Done. I've set the description and excerpt to the same thing because I don't see a need for them to be different things. https://github.com/minetest/blog/pull/154/commits/2011e28c952c6ff3e8a41db32d3d8ef5af4f0a77

Reply:
Maybe, but I would prefer not to fiddle with the theme for this PR.